test/integration/enumerating-databases/list_databases.test.ts

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../../src';
+
+describe('listDatabases', function () {
+  beforeEach(function () {
+    if (process.env.AUTH !== 'auth') {
+      this.skip();
+    }
+  });
+
+  describe('authorizedDatabases flag', function () {
+    const username = 'a';
+    const password = 'b';
+    const mockAuthorizedDb = 'mockAuthorizedDb';
+
+    let client, authorizedClient: MongoClient;
```

Comment:
It's preferable to me at least that we have each variable on its own line, but a little more than just style reasons we're missing the annotation for client.